packages/Toggle/src/toggle.scss

```diff
@@ -1,142 +1,225 @@
-  --ids-toggle-background-checked-disabled: #{tokens.$electric-blue-300};
+    /* Default */
+    --ids-toggle-font-family: #{tokens.$primary-font-family};
+    --ids-toggle-font-size: #{tokens.$font-size-4};
+    --ids-toggle-line-height: normal;
+    --ids-toggle-text-padding: #{tokens.$space-2};
+    --ids-toggle-font-weight: 400;
+    --ids-toggle-width: #{tokens.$space-6};
+    --ids-toggle-height: 2rem;
+    --ids-toggle-background: #{tokens.$grey-300};
+    --ids-toggle-border: transparent;
+    --ids-toggle-transition: background-color 0.3s ease-in-out;
+    --ids-toggle-focus: #{tokens.$focus};
+    --ids-toggle-padding: #{tokens.$space-1};
+    --ids-toggle-font-size-description: #{tokens.$font-size-3};
+    --ids-toggle-line-height-description: normal;
+    --ids-toggle-text-description: #{tokens.$grey-600};
+
+    /* switch */
+    --ids-toggle-width-switch: #{tokens.$space-3};
+    --ids-toggle-height-switch: #{tokens.$space-3};
+    --ids-toggle-margin: 0.2rem;
+    --ids-toggle-background-switch: #{tokens.$samoyed};
+    --ids-toggle-shadow-switch: 0 .1rem .3rem 0 rgb(0 0 0 / 25%);
+
+    /* checked */
+    --ids-toggle-background-checked: #{tokens.$electric-blue-500};
+    --ids-toggle-background-switch-checked: #{tokens.$samoyed};
+    --ids-toggle-border-checked: transparent;
+    --ids-toggle-position-checked:
+        calc(
+            var(--ids-toggle-width) - var(--ids-toggle-width-switch) - 0.4rem
+        );
+
+    /* hover */
+    --ids-toggle-background-hover: #{tokens.$grey-300};
+    --ids-toggle-border-hover: transparent;
+
+    /* disabled */
+    --ids-toggle-background-disabled: #{tokens.$grey-200};
+    --ids-toggle-background-checked-disabled: #{tokens.$electric-blue-300};
+    --ids-toggle-border-disabled: transparent;
+    --ids-toggle-border-checked-disabled: transparent;
+}
+
+[data-brand="workleap"] {
+    /* Default */
+    --ids-toggle-font-family: var(--hop-body-md-medium-font-family);
+    --ids-toggle-font-size: var(--hop-body-md-medium-font-size);
+    --ids-toggle-line-height: var(--hop-body-md-medium-line-height);
+    --ids-toggle-text-padding: var(--hop-space-inline-sm);
+    --ids-toggle-font-weight: var(--hop-body-md-medium-font-weight);
+    --ids-toggle-width: 3rem;
+    --ids-toggle-height: 1.5rem;
+    --ids-toggle-background: var(--hop-neutral-surface);
+    --ids-toggle-border: 0.0625rem solid var(--hop-neutral-border);
+    --ids-toggle-transition: background-color 0.3s ease-in-out;
+    --ids-focus-shadow: 0 0 0 0.12rem var(--hop-neutral-surface), 0 0 0 0.219rem var(--hop-primary-border-focus);
+    --ids-toggle-padding: var(--hop-space-stack-xs);
+    --ids-toggle-font-size-description: var(--hop-body-sm-font-size);
+    --ids-toggle-line-height-description: var(--hop-body-sm-line-height);
+    --ids-toggle-text-description: var(--hop-neutral-text-weak);
+
+    /* switch */
+    --ids-toggle-width-switch: 1rem;
+    --ids-toggle-height-switch: 1rem;
+    --ids-toggle-margin: 0.25rem;
```

Comment:
The igloo one is 0.2rem. For this to be equivalent, it should be 0.125rem. Otherwise you should be using a semantic token here.

Reply:
The Toggle is bigger, according to the design the margin all around is 4px which is 0.25rem

Reply:
after looking at it, it is a larger spacing so I'd probably use var(--hop-space-stack-xs)